### PR TITLE
Pull Request Update 0000-01-23-JCivadelic.md

### PR DESCRIPTION
**Pull request by JCivadelic**
The caption was updated.

Verify caption makes sense.

### CODE REVIEW

File: _posts/0000-01-23-JCivadelic.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Does anyone want to play catch?
 
 ![dodgetocat](https://octodex.github.com/images/dodgetocat_v2.png)
```
Comment (beardofedu) [inline _posts/0000-01-23-JCivadelic.md:8]: I always ❤️ d this Octocat 🔧

**Review by arunnachimuthu — APPROVED** [review]
Lets Play

File: _posts/0000-01-23-JCivadelic.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-23-JCivadelic.md]:
```suggestion
title: "Welcome to JCivadelic's slide!"
```

You should welcome people to your slide 👍

**Review by beardofedu — CHANGES_REQUESTED** [review]
Just a small change before we can 🚢

**Review by beardofedu — APPROVED** [review]

**Comment by JCivadelic** [comment]
#9 @JCivadelic

**Pull request closed without merging**